### ISSUE TITLE
additional details in logging

### ISSUE THREAD

**Issue by danbjoseph** [OpenDroneMap/ODM#745]
i've been using something like:
```
touch /home/ubuntu/projects/canaan/log.txt
sudo chown -R ubuntu /home/ubuntu/projects/canaan/log.txt
./run.sh canaan &>> /home/ubuntu/projects/canaan/log.txt
```

it would be cool if the logged output included additional details:
- ~~all the settings/parameters used during the run~~
- details about the ODM build version?
- system details like memory, etc?

**Comment by smathermather** [comment]
Agreed. As I was doing testing the last couple weeks, having settings and parameters in the log would have been really nice.

**Comment by smathermather** [comment]
Updated with the completed parts crossed out. :)

**Comment by pierotofy** [comment]
This should be implemented by #1297 :pray:
